apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from "react";
+import { View, StyleSheet, Image, Animated } from "react-native";
+import Svg, { Defs, LinearGradient, Stop, Rect } from "react-native-svg";
+import { useTheme } from "@react-navigation/native";
+
+// Number of scrollY pixels to animate 1 -> 0
+const OPACITY_ANIMATION_PERIOD = 200;
+const BLUR_RADIUS = 45;
+
+interface Props {
+  readonly src: string;
+  readonly scrollY: Animated.Value;
+}
+
+const NftViewerBackground: FC<Props> = ({ src, scrollY }) => {
+  const { dark } = useTheme();
+  const gradientColor = dark ? "#131214" : "#fff";
+  const overlayColor = dark ? "#000" : "#fff";
```

Comment:
Probably the same here, there must be a palette color that either it or close to it. If there's not a matching color you should ping design for them to either add a new one (dont do it without asking to not pollute the palette) or choose an existing one.